public/script/workTrolly.js

```diff
@@ -0,0 +1,15 @@
+$("#createNewTask").ready(function() {
+    $("#createNewTask").hide(0);
+    var date = new Date();
+    document.getElementById("creationDateCreateNewTask").innerHTML = date.getDate() + "/" + (date.getMonth()+1) + "/" + date.getFullYear();
```

Comment:
Can we use jquery innerhtml tag to make short

Reply:
yeah i will find the way and implement it.